src/dashboard/tests/test_administration_usage.py

```diff
@@ -17,10 +17,16 @@ def setUp(self):
         helpers.set_setting("dashboard_uuid", "test-uuid")
 
     def test_no_calculation(self):
-        response = self.client.get("/administration/usage/")
-        self.assertFalse(response.context["calculate_usage"])
-        self.assertIn('<a href="?calculate=true"', response.content)
-        self.assertIn("Calculate disk usage", response.content)
+        for calculate in [None, "False", "false", "no", "0", "random"]:
```

Comment:
Parametrization doesn't work in unittest.TestCase subclasses: https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses :(

Reply:
Oh yeah, this is a thing :slightly_frowning_face:  another small thing we can look at in future though :slightly_smiling_face: